docs/src/pages/components/progress/CircularIndeterminate.js

```diff
@@ -1,23 +1,11 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import CircularProgress from '@material-ui/core/CircularProgress';
-
-const useStyles = makeStyles((theme) => ({
-  root: {
-    display: 'flex',
-    '& > * + *': {
-      marginLeft: theme.spacing(2),
-    },
-  },
-}));
+import Box from '@material-ui/core/Box';
 
 export default function CircularIndeterminate() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box sx={{ display: 'flex' }}>
```

Comment:
Do we even need the `Box` now that we are rendering only one element? Or is it added for setting the width?

Reply:
It's not correctly centered without